redis/tests/test_cluster_async.rs

```diff
@@ -779,6 +780,58 @@ fn test_cluster_fan_out_once_even_if_primary_has_multiple_slot_ranges() {
     );
 }
 
+#[test]
```

Comment:
Can we NOT mock this test and possibly test a few different combinations of custom routes/response policies? I know it slows things down further, but I think it's better to actually assert the results of a real server request when feasible.